tsconfig.base.json

```diff
@@ -1,42 +1,39 @@
 {
```

Comment:
I can see a lot of really helpful comments being removed from this file. Can these be placed back in, as they are helpful in-code comments?

Reply:
ok restoring back

Reply:
ok restoring back